Subject:  Fix TBB detection with 2021.2.0
Author:   Orson Teodoro <orsonteodoro@hotmail.com>
Date:     Mon Jun 28 01:29:56 AM PDT 2021 (Unix timestamp: 1624868996)


Error:

-- Found TBB: 2021.2.0 at /usr/lib64/cmake/TBB via TBBConfig.cmake
CMake Error at common/cmake/installTBB.cmake:5 (INSTALL):
  INSTALL TARGETS given target "TBB" which does not exist.
Call Stack (most recent call first):
  common/tasking/CMakeLists.txt:56 (include)


Copyright notice for common/tasking/CMakeLists.txt:

## Copyright 2009-2021 Intel Corporation
## SPDX-License-Identifier: Apache-2.0

diff -urp embree-3.13.0.orig/common/tasking/CMakeLists.txt embree-3.13.0/common/tasking/CMakeLists.txt
--- embree-3.13.0.orig/common/tasking/CMakeLists.txt	2021-06-28 01:25:44.421857128 -0700
+++ embree-3.13.0/common/tasking/CMakeLists.txt	2021-06-28 01:26:54.948604027 -0700
@@ -1,6 +1,8 @@
 ## Copyright 2009-2021 Intel Corporation
 ## SPDX-License-Identifier: Apache-2.0
 
+# oteodoro - changes to make it work with TBB 2021.2
+
 IF (TASKING_INTERNAL)
   ADD_LIBRARY(tasking STATIC taskschedulerinternal.cpp)
 ELSEIF (TASKING_TBB)
@@ -21,7 +23,7 @@ ELSEIF (TASKING_TBB)
   else()
     # Try getting TBB via config first
     find_package(TBB 2021 COMPONENTS tbb CONFIG ${TBB_FIND_PACKAGE_OPTION})
-    if (TBB_FOUND)
+    if (TBB_FOUND AND FALSE)								# oteodoro - disabled conditional block
       TARGET_LINK_LIBRARIES(tasking PUBLIC TBB::tbb)
       message("-- Found TBB: ${TBB_VERSION} at ${TBB_DIR} via TBBConfig.cmake")
     else()
